fcli-ssc/src/main/java/com/fortify/cli/ssc/role/cli/cmd/SSCRoleCommands.java

```diff
@@ -0,0 +1,17 @@
+package com.fortify.cli.ssc.role.cli.cmd;
+
+import picocli.CommandLine.Command;
+
+@Command(
+        name = "role",
+        aliases = {},
+        subcommands = {
+                SSCRoleListCommand.class,
```

Comment:
Can you please put the subcommands in alphabetical order?

Reply:
Done.